tests/models/test_pytorch_models.py

```diff
@@ -165,38 +163,43 @@ def test_file_properties(self, tmpdir, net, optimizer):
         )
         assert tiledb_obj._file_properties["TILEDB_ML_MODEL_PREVIEW"] == str(model)
 
-    def test_tensorboard_callback_meta(self, tmpdir, net, optimizer, mocker):
+    @net
+    def test_tensorboard_callback_meta(self, tmpdir, net):
         model = net()
         tiledb_array = os.path.join(tmpdir, "model_array")
         tiledb_obj = PyTorchTileDBModel(uri=tiledb_array, model=model)
 
-        mocker.patch(
-            "tiledb.ml.models.pytorch.PyTorchTileDBModel._get_tensorboard_files",
-            return_value={
-                f"{tmpdir}/event_file_name_1": b"test_bytes_1",
-                f"{tmpdir}/event_file_name_2": b"test_bytes_2",
-            },
-        )
+        # SummaryWriter creates file(s) under log_dir
+        log_dir = os.path.join(tmpdir, "logs")
+        writer = SummaryWriter(log_dir=log_dir)
+        log_files = read_files(log_dir)
+        assert log_files
 
-        writer = SummaryWriter()
         tiledb_obj.save(update=False, summary_writer=writer)
-
         with tiledb.open(tiledb_array) as A:
-            assert len(pickle.loads(A.meta["__TENSORBOARD__"])) == 2
-            assert pickle.loads(A.meta["__TENSORBOARD__"]) == {
-                f"{tmpdir}/event_file_name_1": b"test_bytes_1",
-                f"{tmpdir}/event_file_name_2": b"test_bytes_2",
-            }
+            assert pickle.loads(A.meta["__TENSORBOARD__"]) == log_files
+        shutil.rmtree(log_dir)
```

Comment:
Do you need that? Since the folder is under `tmpdir` do you really need to `rm` it with `shutil`?

Reply:
The tmpdir is automatically cleaned up after the test returns; here we want to clean up `log_dir` to verify that `load()` recreates the files.